test/test_axes.jl

```diff
@@ -63,3 +63,17 @@ end
     @test twpl[:top_margin] == 2Plots.cm
     @test twpl[:bottom_margin] == 2Plots.cm
 end
+
+@testset "aliases" begin
+    plot(1:2, xl = "x label")
```

Comment:
```suggestion
    @test plot(1:2, xl = "x label")[1][:xaxis][:guide] == "x label"
```
And the others alike

Reply:
Woops, incomplete. Thanks.